cirq/contrib/quimb/mps_simulator.py

```diff
@@ -354,9 +368,8 @@ def __init__(
             n = self.grouping[qubit]
             self.M[n] @= qtn.Tensor(x, inds=(self.i_str(i),))
             initial_state = initial_state // d
-        self.rsum2_cutoff = rsum2_cutoff
-        self.sum_prob_atol = sum_prob_atol
-        self.num_svd_splits = 0
+        self.simulation_options = simulation_options
+        self.e_n_s: List[float] = []
```

Comment:
Is there a more descriptive name we could use here (e.g. expand the acronym)? Otherwise a comment describing what this is would be helpful.

Reply:
Named in `estimated_gate_error_list`.